pkg/archive/archive.go

```diff
@@ -395,6 +395,23 @@ func ReadSecurityXattrToTarHeader(path string, hdr *tar.Header) error {
 	return nil
 }
 
+// ReadUserXattrToTarHeader reads user.* xattr from filesystem to a tar header
+func ReadUserXattrToTarHeader(path string, hdr *tar.Header) error {
+	if xattrs, _ := system.Llistxattr(path); xattrs != nil {
+		for _, key := range xattrs {
+			if strings.HasPrefix(key, "user.") {
+				if value, _ := system.Lgetxattr(path, key); value != nil {
```

Comment:
if we get here, xattrs are supported so I think we should not ignore errors

Reply:
I've intentionally made it silently ignore errors in order to be consistent with existing behavior and usage of the ReadSecurityXattrToTarHeader function. That said, I do prefer stricter error handling. I suppose we should also update behavior and usage of the ReadSecurityXattrToTarHeader function, for consistency.

Reply:
Fixed in 0da4bc60b3babedb098dd7a75ae4ec0b75031eab.